src/index.js

```diff
@@ -166,6 +167,38 @@ class SnykClient {
   }
 
 
+  /**
+   * "List All Aggregated Issues"
+   * @param {string} orgId - Organization ID
+   * @param {string} projectId - Project ID
+   * @param {Object} filters - Object with filters
+   * @returns {Object} object representing the list of issues
+   */
+   async listAggregatedIssues (orgId, projectId, filters = {}) {
+    return retry(
+      async () => {
+        return this.snykRequest({
+          method: 'POST',
+          uri: `/org/${orgId}/project/${projectId}/aggregated-issues`,
+          body: filters
+        });
+      },
+      {
+        delay: 5000,
+        factor: 1.2,
+        maxAttempts: this.retries,
+        handleError(err, context) {
+          const code = err.statusCode;
+          console.log({err});
```

Comment:
Do we actually want the console log here?

Reply:
I saw this in the current `listIssues` method. Probably not. I can remove before releasing.